spec/springboard/client/resource_spec.rb

```diff
@@ -97,6 +97,17 @@
     end
   end
 
+  describe "only" do
+    it "should set the _only parameter based on the given values" do
+      expect(resource.only('f1', 'f2').uri.query).to eq('_only[]=f1&_only[]=f2')
+    end
+
+    it "should replace the existing _only parameters" do
+      resource.only('f1')
```

Comment:
This test doesn't actually test anything because it's not using the resource returned by the first call to `only`. Should be something like:

```ruby
expect(resource.only('f1').only('f2', 'f3').uri.query).to eq('_only[]=f2&_only[]=f3')
```

Reply:
You are right, I completely missed that. Fixed the test.